.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["standard", "prettier"],
+  "plugins": ["react", "prettier"]
+}
```

Comment:
use this rules, instead: https://github.com/entria/ReactNavigationRelayModern/blob/master/.eslintrc